src/instance.ts

```diff
@@ -355,7 +355,9 @@ class Instance extends common.GrpcServiceObject {
         let backupInstances: Backup[] | null = null;
         if (backups) {
           backupInstances = backups.map(backup => {
-            return this.backup(backup.name!);
+            const backupInstance = this.backup(backup.name!);
+            backupInstance.metadata = backup;
```

Comment:
Nice catch. Please add a test to see that this is set.

Reply:
I believe I added that verification in an existing [test](https://github.com/googleapis/nodejs-spanner/pull/1198/files#diff-189d5720e2e29b973f957a72855ced1cR1459). Is it sufficient?

Reply:
🤦 sorry. LG!